docs/docs/getting-started/v3-migration.md

```diff
@@ -64,6 +64,9 @@ A number of changes were made to the configuration options passed to the `Chart`
 #### Specific changes
 
 * `hover.animationDuration` is now configured in `animation.active.duration`
+* `tooltips.titleFontFamily` is now configured in `tooltips.titleFont`
```

Comment:
Not sure how this would be best worded out. This applies also to *FontColor, *FontSize, *FontStyle and *LineHeight everywhere.

Reply:
We surely need to list them somewhere, so people doesn't have to dig into source code to figure that out. Could move it under #Tooltip instead of #Options.  Or under #Default section, below to `*FontColor, *FontSize, *FontStyle and *LineHeigh` changes. For example ` options.tooltips.titleFontFamily` was renamed to `options.tooltips.titleFont`